### DISCUSSION TITLE
Support "Guest Authors"

### DISCUSSION THREAD

**Discussion by pkfrank** [forem/forem#15325] [Feature Requests]
### **Is your feature request related to a problem? Please describe.**
As an org admin, I would like to invite guest authors to submit articles into my organization without adding them as a full-fledged member.

### **Describe the solution you'd like**
- Org admins can assign the "role" of org members (admin, normal, guest)
- Guest Authors can publish into an org like a regular org member
- The org profile page will have a divider between full org members and guest authors
- Appropriate distinguishing design on the article page to show it's a guest post

**Stretch Features**
- Org admins can provide an invite token that automatically invites member in as Guest Author
- Org admins can enable an optional "approval flow" for publishing posts 
    - This would require an org admin to "approve" the post before it goes live

**Comment by mbifulco** [comment]
@pkfrank thank you for this! I've got a suggestion for what might be a simpler scenario - for example, this [guest post on the Gymnasium org](https://dev.to/gymnasium/from-filler-to-matter-wireframes-and-prototypes-in-ux-design-4ogf) was created by a contract writer we work with who is unlikely to ever have a dev.to account, particularly since this is a cross-post from Medium.  It'd be great to be able to crosspost his articles myself, and switch the byline to his name and headshot somehow.

I've done my best to give him credit at the end of the article, but it still feels a bit disingenuous 🤔

**Comment by rviscomi** [comment]
+1 I think this would be a great feature! I'm a member of the [Google Web Dev](https://dev.to/chromiumdev) and [HTTP Archive](https://dev.to/httparchive) orgs and the ability to allow guest posts would really help broaden the orgs' perspectives.

I can also imagine a different approach where an org member can take any public post and ask the author's permission to grant co-ownership to their org, making it a guest post. Either the author or the org members should have write access to the post and can break the agreement at any time to revert the post back to personal. WDYT?

**Comment by michael-tharrington** [comment]
This feature would be an excellent solution for the problem when an org writer leaves an org, but we still want to credit both the org and writer on the post. 

If this was in place, the org admin could just change the person's org status from member to guest.
